simulation/runSim.sh

```diff
@@ -1,8 +1,8 @@
-COPPELIA_SIM_SCRIPT=~/dev/coppelia/coppeliaSim.sh
+COPPELIA_SIM_SCRIPT=~/docker-build/coppelia/coppeliaSim.sh
```

Comment:
Do you want this script in ~/dev/coppelia/coppeliaSim.sh? Should be an easy fix in the dockerfile if you want to move it.

Reply:
I don't have any opinion - both work. The entire coppelia folder would need to be moved, but that shouldn't be difficult.